_pages/attributes.md

```diff
@@ -8,6 +8,10 @@ Here are the possible attributes that can be requested at a given IAL. This tabl
 
 It is important to expect any number of characters in the `(string)` datatype unless directly followed by a number such as `(string36)`. Strings are encrypted and stored in a text datatype with a maximum length of 65,535 bytes.
 
+{% capture checkmark %}
+ {% include icon.html style="text-blue" icon_name="check" %}
```

Comment:
oh note to me to see what i need to do to make this accessible -- these checkmarks have important semantic meeting